.github/workflows/ci-unix-static.yml

```diff
@@ -42,11 +42,16 @@ jobs:
         profile: minimal
         toolchain: stable
         override: true
+    - name: Install dependencies (Linux)
+      if: runner.os == 'Linux'
+      run: |
+        sudo apt update
+        sudo apt install imagemagick libjpeg-turbo8-dev libpng-dev 
     - name: Install dependencies (macOS)
       if: runner.os == 'macOS'
       run: brew install imagemagick
     - name: Install cargo-c (linux)
-      if: matrix.os == 'ubuntu-latest'
+      if: matrix.os == 'ubuntu-24.04'
```

Comment:
Nit: we can test `runner.os == 'Linux'` instead.

Reply:
Done